kamailio/traffic-filter-role.cfg

```diff
@@ -20,6 +20,19 @@ route[FILTER_REQUEST]
 
 route[FILTER_REQUEST_DOMAIN]
 {
+    #!ifdef DISPATCHER_ROLE
+    if (
```

Comment:
Hello @k-anderson 
Purpose of this code to resolve HOLD/UNHOL issue when FreeSwitch located ad different IP then Kamailio.
Please look pcap files KAZOO-5182.

When call is established, then SIP phone send REINVITE to with FreeSwitch contact in RURI header.
Is not possible to set `trusted` fro everyone sip device.
Required to pass REINVITE with IP in RURI header if this IP is FreeSwitch contact.

This code is do it.